### ISSUE TITLE
[Visual Bug] Z-Fighting with certain blocks

### ISSUE THREAD

**Issue by dbrighthd** [sp614x/optifine#4759]
If an Iron bar is placed on the ground, there is a lot of z-fighting with the block under it and the base of the iron bra

Reproduce:
 place an iron bar on the ground and move your mouse around (no shaders or texture packs)

I am not sure if something in my settings are causing this so just in case here are my optionsof & options files
[options.txt](https://github.com/sp614x/optifine/files/5160203/options.txt)
[optionsof.txt](https://github.com/sp614x/optifine/files/5160201/optionsof.txt)

Optifine version is 1.16.2_HD_U_G3
Screenshots:
![2020-09-01_22 05 34](https://user-images.githubusercontent.com/68525639/91934304-e7bb7a00-ec9f-11ea-8c38-060ed743c90b.png)
![2020-09-01_22 05 35](https://user-images.githubusercontent.com/68525639/91934308-e9853d80-ec9f-11ea-8a38-a452b559a0dc.png)

This only happens with optifine. It looks fine with vanilla 1.16.2

This also happens with shadows occasionally, but it is harder to reproduce and is less noticeable

**Comment by Cart3r1234** [comment]
I've had it happen in vanilla, there are actually a few bug reports on the mojang bug tracker that is open, including this one https://bugs.mojang.com/browse/MC-151004

**Comment by dbrighthd** [comment]
> I've had it happen in vanilla, there are actually a few bug reports on the mojang bug tracker that is open, including this one https://bugs.mojang.com/browse/MC-151004

Oh well for me there is no Z fighting in vanilla 1.16.2, might be a different case for other people

**Comment by Cart3r1234** [comment]
Yeah the fact that you can zoom makes it a lot more apparent while using optifine, as you can get farther away and still see it, as it doesn't happen as much at closer ranges.

**Comment by sp614x** [comment]
Does it happen when Render Regions is disabled?

**Comment by Cart3r1234** [comment]
Yeah I never turned render regions on and it still happens for me in both optifine 1.16.1/.2 and in vanilla 1.16.1/.2

**Comment by dbrighthd** [comment]
> Does it happen when Render Regions is disabled?

Oh apparently not.
Disabling render regions fixes it for me, thanks.


> Yeah I never turned render regions on and it still happens for me in both optifine 1.16.1/.2 and in vanilla 1.16.1/.2

For me there is no Z fighting in vanilla, only with optifine
